.github/CODEOWNERS

```diff
@@ -6,4 +6,4 @@
 cli/*   @shaunmitchellve
 
 # Solution Reviewers
-solutions/* @davelanglois-ssc @alaincormier-ssc
\ No newline at end of file
+solutions/* @davelanglois-ssc @alaincormier-ssc @lucstjean-ssc @Michael-McAuley-SSC @amcmullin @CraigBaltzer uday-ssc ovidiuasiminei-ssc stephanemillaire-ssc
```

Comment:
looks like the last 3 users are missing the `@` in front

Reply:
Thanks for catching that! I was fighting some vscode hotkey function when I added those last 3.